docs/en/05_Contributing/14_PHP_Coding_Conventions.md

```diff
@@ -329,6 +329,18 @@ Statements with `if` can be written without braces on a single line as the block
 	if($a == $b) doThis();
 	else doThat();
 
+#### Prefer identical (===) comparisons over equality (==)
+
+Where possible, use type-strict identical comparisons instead of loosely typed equality comparisons.
+Read more in the PHP documentation for [comparison operators](http://php.net/manual/en/language.operators.comparison.php) and [object comparison](http://php.net/manual/en/language.oop5.object-comparison.php).
+
+	:::php
+	// good - only need to cast to (int) if $a might not already be an int
+	if((int)$a === 100) doThis();
```

Comment:
Also if we do want to follow PSR-2 in code examples there should be a space after the control keyword (if) http://www.php-fig.org/psr/psr-2/#5-control-structures